src/main/java/com/zs/exc5/Product.java

```diff
@@ -0,0 +1,173 @@
+package main.java.com.zs.exc5;
+
+import java.sql.*;
+import java.util.LinkedHashSet;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.logging.Logger;
+
+/**
+ * This class is created to perform the operations in product table.
+ */
+public class Product {
+    private Connection c = null;
+    private PreparedStatement stmt = null;
+    private ResultSet rs;
+    private String pid, pname;
+    int pquantity, pval;
+    Scanner scan = new Scanner(System.in);
+    private static Logger logr;
+    static {
+        System.setProperty("java.util.logging.config.file",
+                "/home/raramuri/IdeaProjects/Myproject/src/logger/logging.properties");
+
+        logr = Logger.getLogger(Ecommerce.class.getName());
+    }
+    /**
+     * To create a database connection.
+     */
+    public void connection() throws ClassNotFoundException, SQLException {
+
+        try {
+            Class.forName("org.postgresql.Driver");
```

Comment:
instead of writing the connection statement multiple number of times can you write it in a separate class and use it everywhere.